tests/unit/cli/test_review_command.py

```
@@ -131,6 +134,70 @@ def test_review_json_output_echoes_payload() -> None:
+    )
+    mock_config.review.depth = 1
+    mock_config.review.strictness = ReviewStrictness.BALANCED
+    mock_config.review.sensitivity = MagicMock()
+    mock_config.review.force_semantic_chunking = False
+    mock_config.review.checklist_display = ChecklistDisplay.OFF
+
+    with patch("lintro.cli_utils.commands.review.require_ai"):
+        with patch(
+            "lintro.cli_utils.commands.review.get_config",
+            return_value=mock_config,
+        ):
```

Ruff (0.15.20)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)